packages/fluentui/react-northstar/src/themes/teams/components/Datepicker/datepickerCalendarHeaderStyles.ts

```diff
@@ -0,0 +1,16 @@
+import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+import { DatepickerVariables } from './datepickerVariables';
+import { DatepickerCalendarHeaderStylesProps } from '../../../../components/Datepicker/DatepickerCalendarHeader';
+
+export const datepickerCalendarHeaderStyles: ComponentSlotStylesPrepared<
+  DatepickerCalendarHeaderStylesProps,
+  DatepickerVariables
+> = {
+  root: (): ICSSInJSStyle => {
+    return {
+      display: 'grid',
+      gridTemplateColumns: '1fr auto auto',
+      msGridColumns: '1fr auto auto',
```

Comment:
Just a reminder, tho we are stopped in working with IE11 for now... If using display grid each cell should receive the placement to be compatible with IE11

Reply:
Oh, good catch!

Reply:
Not sure if we leave this `display: grid` after the styling work is done, but I'll add a comment there to check it then